brambling/urls.py

```diff
@@ -76,8 +76,6 @@
     FinancesView,
 )
 from brambling.views.payment import (
-    DwollaConnectView,
-    DwollaWebhookView,
     StripeWebhookView,
 )
```

Comment:
Wanna clean this import up while you're at it? 😇

Reply:
done